assets/css/components/_tray.sass

```diff
@@ -26,19 +29,41 @@ body.thesis-tray-open
     width          : 6px
     z-index        : 10
     pointer-events : none
-    +linear-gradient(left, rgba(black, .2), white)
-    background-color : white
-
-  .attribution
-    position    : absolute
-    bottom      : 0
-    left        : 0
-    width       : 100%
-    line-height : 40px
-    font-size   : 13px
-    text-align  : center
-    font-weight : 300
-    background  : white
+    +linear-gradient(left, rgba(black, .2), rgba(white, 0))
+    background-color: transparent
+
+  .thesis-tray-notifications
+    width:      100%
+
+    .thesis-tray-notifications-wrap
+      background: $zahri_pink
+      padding:    15px 40px 15px 25px
+
+    .thesis-tray-notification
+      margin-bottom: 12px
+      color:         white
+      font-size:     12px
+      position:      relative
+      padding-left:  30px
+
+      i
+        font-family: $fontawesome
+        color:       white
+        font-size:   14px
+        position:    absolute
+        top:         1px
+        left:        0
+
+      &:last-child
+        margin-bottom: 0
+
+  .thesis-attribution
+    width:       100%
+    line-height: 40px
+    font-size:   13px
+    text-align:  center
+    font-weight: 300
+    background:  white
```

Comment:
Some tweaks here to account for the new Notifications section above the tray content.